keyserver/README.md

```diff
@@ -0,0 +1,56 @@
+# CONIKS Server implementation in Golang
+
+## Usage
+```
+⇒  go install github.com/coniks-sys/coniks-go/keyserver/coniksserver
+⇒  coniksserver -h
+/----------------------------------------------\
+|                  ,,:::::,:,,                 |
+|             :irrsrrrrrrrrrrrsrr:,            |
+|          irsrr::::,::::::::::iirrsi,         |
+|       ,rsr;:::irss5SXX9SXH2ssi:,:irrs:       |
+|      rrri::rsX9GGG99XXS9X99GGG9Hr:,:;rr:     |
+|    ,sri::r9GG9X252SSXXXXXSS25H99M9s::irri    |
+|    sr;::5M99H259MBB@@@B@B@BMGX2HS9GGr:i;ri   |
+|   rr;::XMXXH5G@B@BBMBMBMBM@B@BBX5SX9Mr:irs,  |
+|   sri:sMXXS5M@BBMBMMMMMMMBMBM@B@SSS9GG:iis:  |
+|   sii:2G9SS5@BBMMMMMMMBMMMMMBMBB92XS9Gi:rri  |
+|   rri:iM995HM@MBMMMBMMMBMBMBB@BM5SS9M5,iis,  |
+|   :s;i:rMGX5@BMMMM@B@B@B@@@B@MX2SX9MX::irr   |
+|    :ri::iSG9B@BBM@GGGMMMGG9X222X9M9s,:irs    |
+|     riii:,r2@B@B@B9252525252SS99Xs:,irsr     |
+|    ;riiiii:,:s5MB@M9GGGGXXSXS9H:,::rrs:      |
+|    sri;irrrii,,,::iirrriX9XS9GS,;rsr:        |
+|   :srrrsrr:irrrrii:i:::,9MGMM@Mii:           |
+|   2sr::,      ,:iirrrrriMMGSsr:              |
+|  ,,                                          |
+\----------------------------------------------/
+
+Usage:
+  coniksserver [command]
+
+Available Commands:
+  init        Create a configuration file and generate all keys
+  run         Run a CONIKS server instance
+
+Flags:
+  -h, --help   help for coniksserver
+
+Use "coniksserver [command] --help" for more information about a command.
+```
+
+Run the server
+```
+⇒  mkdir coniks; cd coniks
+⇒  coniksserver init -c // creates all files including a self-signed ssl keys/cert
+⇒  coniksserver run     // run
+```
+
+You can reload the server's policies while it's running by editing the `config.toml` file
+and possibly replace `vrf.priv` with a new key, then run
+```
+⇒  kill -USR2 pid
```

Comment:
Maybe I thought that' not a big deal, so I gave it a try in d144f97cbe7f0ea300954560108ddea0ff5f9427.